vendor/swipebox

```diff
@@ -0,0 +1 @@
+Subproject commit 25defbe5a5870657432208ed985d09bfef4d7d67
```

Comment:
Looks like the submodule may not have been set up right?

![screen shot 2015-05-26 at 8 41 25 pm](https://cloud.githubusercontent.com/assets/86842/7826418/9d8f72b6-03e7-11e5-9d3c-86c0a5813ade.png)